app/views/news_feed/index.html.erb

```diff
@@ -3,6 +3,6 @@
 <% @activities.each do |activity| %>
   <div class = "activity">
   [username here]<%= link_to activity.owner.name, activity.owner if activity.owner %>
-  posted <%= link_to activity.trackable.summary, post_path(activity.trackable) %>
+  <%= activity.trackable_type %>ed <%= link_to activity.trackable.summary, post_path(activity.trackable) %>
```

Comment:
is the 'ed' here important?

Reply:
It's  just so it reads in English. The type returns 'comment' or 'post'

Reply:
feels like this should be a helper method that returns the past tense of a verb